notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -740,6 +797,24 @@ export default {
       this.postingNote = false;
       this.postKey++;
     },
+    isDefaultContent(noteContent) {
+      const div = document.createElement('div');
+      div.innerHTML = noteContent;
+      if ( div.childElementCount === 2) {
+        const childrenWrapper = CKEDITOR.instances['notesContent'].window.$.document.getElementById('note-children-container');
+        if ( childrenWrapper ) {
```

Comment:
Can be simplified to:
return childrenWrapper && childrenWrapper.nextElementSibling.innerText.trim().length === 0 ? true : false ;